company-box-icons.el

```diff
@@ -98,12 +98,30 @@ Each element have the form:
 (KIND . ICON)
 
 Where KIND correspond to a number, the CompletionItemKind from the LSP [1]
-
+a
 See `company-box-icons-functions' for the ICON format.
 
 [1] https://github.com/Microsoft/language-server-protocol/blob/gh-pages/\
 specification.md#completion-request-leftwards_arrow_with_hook.")
 
+(defvar company-box-icons-acphp
+  (eval-when-compile
+    `(,(company-box-icons-image "Interface.png")
+      ,(company-box-icons-image "Class.png")
+      ,(company-box-icons-image "Method.png")
+      ,(company-box-icons-image "Method.png")
+      ,(company-box-icons-image "Property.png")
+      ,(company-box-icons-image "Constant.png")
+      ,(company-box-icons-image "Field.png")
+      ,(company-box-icons-image "Interface.png")
+      ,(company-box-icons-image "Namespace.png")
+      ,(company-box-icons-image "Template.png")
+      ,(company-box-icons-image "Misc.png")))
+  "List of icons to use with PHP candidates.
+The list has the form:
+(FUNCTION VALUE FEATURE FACE).
```

Comment:
Can you change the docstring with `(TRAIT CLASS METHOD FUNTION....)`

Reply:
I did not really get your meaning here -- see my last commit if that addresses you concern.